UnityProject/Assets/Scripts/FluidPipe/Filter.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using Atmospherics;
+using UnityEngine;
+
+namespace Pipes
+{
+	public class Filter : MonoPipe
+	{
+		public SpriteHandler spriteHandlerOverlay = null;
```

Comment:
Same thing. Make it private with a public getter and  [SerializeField] decorator.